.circleci/config.yml

```diff
@@ -311,6 +311,23 @@ commands:
       - run:
           command: ./.circleci/report-coverage.sh << parameters.list >>
 
+  run-playwright-tests:
+    parameters:
+      project:
+        type: string
+    steps:
+      - run:
+          name: Running Playwright tests
+          # Supports 'Re-run failed tests only'. See this for more info: https://circleci.com/docs/rerun-failed-tests-only/
+          command: |
+            cd packages/functional-tests
+            TEST_FILES=$(circleci tests glob "tests/**/*.spec.ts")
+            echo $TEST_FILES | circleci tests run --command="xargs yarn playwright test --project=<< parameters.project >>" --verbose --split-by=timings
```

Comment:
I vaguely remember that @vbudhram did something similar to this by splitting the tests by timing and it failed a lot of tests, not sure exactly why. But we can give it a try and monitor the tests for performance.

Reply:
There was a lot of other flakiness at that time. I didn't see any issues this time around, but thank you for calling it out. Definitely good to note this. 👍